src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -268,13 +271,26 @@ class Chart extends React.Component<Props> {
     const loading = isLoading;
     const reloading = isLoading;
 
+    const yMax = Math.max(
```

Comment:
Both `Math.max` and `Math.min` can take variable arguments. So you can do it compute the min and max each with one call.

Reply:
I was mixing max/min before and forgot to fix this, thanks!